projects/opendr_ws/src/perception/scripts/panoptic_segmentation_efficient_ps.py

```diff
@@ -134,9 +152,11 @@ def callback(self, data: ROS_Image):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('checkpoint', type=str, help='load the model weights from the provided path')
     parser.add_argument('image_topic', type=str, help='listen to images on this topic')
-    parser.add_argument('--heatmap_topic', type=str, help='publish the semantic and instance maps on this topic')
+    parser.add_argument('checkpoint', type=str,
+                        help='download pretrained models [cityscapes, kitti] or load from the provided path')
+    parser.add_argument('--heatmap_topic', type=str,
+                        help='publish the semantic and instance maps on this topic')
     parser.add_argument('--visualization_topic', type=str,
```

Comment:
This could act similar to other topics and not publish anything if the topic is `None` and not use a separate flag.

Note that there is an issue with all other nodes i fixed, where the user is not able to provide `None` values through argparse, but i intend to fix this later, and will probably `sweep-fix` it across all toolkit ROS1/2 nodes as they should be technically the same.

Reply:
I dug deeper in the code and found out this is indeed a flag for the learner's visualize method and not what i thought before. Visualization topic seems to work as expected. Nevertheless, do you think we can remove it altogether and pass a True value to the visualization method so we get the maximum information out of it and simplify the interface?

As a side note, visualization seems to not work with different sizes of input images, can you confirm this? I try to run the node and provide images  through a 640x480 webcam and i get a `ValueError: images do not match` error on `...efficient_ps/efficient_ps_learner.py", line 547, in visualize panoptics_img = PilImage.blend(`...`.